### ISSUE TITLE
Create Local Rule for Microsoft-Windows-SmartScreen/Debug

### ISSUE THREAD

**Issue by Aldrims** [wazuh/wazuh-ruleset#834]
I receive log on my manager coming from a windows  wazuh agent 
Wazuh Archives:
![Image_github](https://user-images.githubusercontent.com/46164298/122054212-c15a6b80-cde7-11eb-8691-88e0fcb5ece4.png)
I tried to create some rules for it trying to do the same architecture as wazuh existing rules for windows
Local Rules
![image](https://user-images.githubusercontent.com/46164298/122054346-e2bb5780-cde7-11eb-80dc-bf71a08d0c6f.png)
I still can't have any alert popping

**Comment by Nekaze** [comment]
Hello @Aldrims! I hope you're having a great day.

I have reviewed your issue and it appears that you are trying to consume a channel that is not subscriptable. I myself have experienced exactly the [same issue not long ago](https://github.com/wazuh/wazuh/issues/8395#issuecomment-834714497) trying to consume the Microsoft-Windows-WMI-Activity/Trace channel. Please refer to my comments there for more detailed information.

Unfortunately, Debug channels cannot be subscribed to. This is a limitation on Microsoft side. From [EvtSubscribe function documentation](https://docs.microsoft.com/en-us/windows/win32/api/winevt/nf-winevt-evtsubscribe) :
> ChannelPath
> The name of the Admin or Operational channel that contains the events that you want to subscribe to **(you cannot subscribe to Analytic or Debug channels)**. The path is required if the Query parameter contains an XPath query; the path is ignored if the Query parameter contains a structured XML query.

Another thing I attempted was to consume the log file directly, but the file is an .etl file so wazuh cannot process it directly.

To sum up, what you want to achieve is just not supported :( . Nevertheless, you're very welcome to create a feature request on [wazuh/wazuh](https://github.com/wazuh/wazuh) repository!

I hope this helps!
Best regards

**Comment by Aldrims** [comment]
The difference is that like Microsoft-Windows-WMI-Activity/Operation I got some log  in the wazuh archive like I show it on the first screen.

**Comment by Nekaze** [comment]
Are you sure they are coming from Microsoft-Windows-SmartScreen/Debug provider? Can you share the full event? I believe those logs you are getting are most likely from other providers such as Microsoft-Windows-Windows Defender/Operational or .NET Runtime.

Wazuh currently has rules 61019 and 62107 to address this scenarios.

Hope this helps!
Kind regards

**Comment by Aldrims** [comment]
![image_github2](https://user-images.githubusercontent.com/46164298/122096412-d1d20c80-ce0e-11eb-98c1-04355d70795b.png)

**Comment by Nekaze** [comment]
This seems to be a hardcoded log in wazuh-logtest. Could you share the full wazuh event as you get it in the wazuh archives? Like the one you showed from Kibana, in the first picture. Also, could you share the ossec.log from the agent where you are getting this events?
Thanks!

**Comment by Aldrims** [comment]
![Image_github3](https://user-images.githubusercontent.com/46164298/122099562-6db14780-ce12-11eb-9b3d-f1b1c7b867f1.png)
![image_github4](https://user-images.githubusercontent.com/46164298/122099796-b49f3d00-ce12-11eb-9fc3-926ee9b69de2.png)
![image](https://user-images.githubusercontent.com/46164298/122100363-62125080-ce13-11eb-883c-35029f56025b.png)

**Comment by Aldrims** [comment]
Weirdly it work when writing my rules directly in the ruleset instead of my local file xml

**Comment by Nekaze** [comment]
I'm glad you were able to sort it out!
Best regards

**Comment by Nekaze** [comment]
I understand your issue has been resolved, would you mind closing the issue? 
Thanks!
Kind regards

**Comment by Aldrims** [comment]
My issue is not really resolved because I should not use the /var/ossec/ruleset/rules  for custom rules but /var/ossec/etc/rules/

**Comment by Nekaze** [comment]
I see... sorry for the missunderstanding. Local rules are loaded last, so It is possible that there´s another rule hitting (and maybe muting) the log you are testing. That´s why putting it in the default rules might seem to work, yet it is changing evaluation order. If you could share the log in text format I could perform some tests myself and see what is going on

Best regards

**Comment by Aldrims** [comment]
`{"win":{"system":{"providerName":"Microsoft-Windows-SmartScreen","providerGuid":"{3cb2a168-fe34-4a4e-bdad-dcf422f34473}","eventID":"1000","version":"0","level":"4","task":"0","opcode":"0","keywords":"0x8000000000000000","systemTime":"2021-06-23T10:06:10.8639688Z","eventRecordID":"188","processID":"10072","threadID":"9164","channel":"Microsoft-Windows-SmartScreen/Debug","computer":"XXXXXX","severityValue":"INFORMATION","message":"\"Événement SmartScreen de l'application\""},"eventdata":{"filePath":"XXXX.exe","fullFileHash":"XXXXX=","markOfTheWeb":"{\\\"HostUrl\\\":\\\"https://www.URL/EXE.exe\\\",\\\"ReferrerUrl\\\":\\\"URL/\\\",\\\"ZoneId\\\":\\\"3\\\"}","callingProcessId":"4120","callingProcessCreationTime":"132682353642833500","enforcement":"warn","experience":"Untrusted"}}}`

**Comment by jnasselle** [comment]
Hi @Aldrims,

I'm going to move this issue since this repository was absorbed by [wazuh/wazuh](https://github.com/wazuh/wazuh) repo under `ruleset` folder, deprecating this one.

In the meantime, could you please tell me your Wazuh Manager Version and the current permissions of files located in `/var/ossec/etc/rules` directory?

Regards,
Nico
